src/Middleware/HttpsPolicy/src/HstsOptions.cs

```diff
@@ -40,7 +40,7 @@ public class HstsOptions
         /// <summary>
         /// A list of host names that will not add the HSTS header.
         /// </summary>
-        public IList<string> ExcludedHosts { get; } = new List<string>
+        public IList<string> ExcludedHosts { get; } = new List<string>(3)
```

Comment:
For reference https://github.com/dotnet/roslyn/issues/43944

Reply:
I was wondering if there was a general solution to what's being done in this PR, seems like that's the appropriate discussion point for the general case.